modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -1061,13 +1120,79 @@ definition {
 			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU");
 	}
 
+	@description = "Verify buyers cannot change the currency of a channel."
+	@priority = "3"
+	test BuyersCannotEditChannelCurrency {
```

Comment:
@Kvale1733 in accordance with @victoroscuro, please change the name of this test to this:
```suggestion
	@description = "Verify users without edit permission cannot change the currency of a channel."
	@priority = "3"
	test CanNotEditChannelCurrencyAsUserWithoutEditPermission {
```